mpu6050_i2c_pi_ros_driver:
  ros__parameters:
    frame_id: "imu"
    publish_frequency: 20.0
    lin_acc_offset_x: 1.660977289193059
    lin_acc_offset_y: 0.19307000053888004
    lin_acc_offset_z: 0.5455802841694926
    ang_vel_offset_x: -0.011385502460200678
    ang_vel_offset_y: 0.0017311290290363604
    ang_vel_offset_z: -6.658188573216775e-05